Source/WebKit/Shared/UpdateInfo.h

```diff
@@ -68,7 +68,7 @@ class UpdateInfo {
     float updateScaleFactor { 0 };
 
     // The handle of the shareable bitmap containing the updates. Will be null if there are no updates.
-    ShareableBitmap::Handle bitmapHandle;
+    mutable ShareableBitmap::Handle bitmapHandle;
```

Comment:
I think, here is the same problem:
Here you make this mutable, and there's a const UpdateInfo& receiver that then clobbers the instance and makes the instance "it kind of is ok, but it really is broken as the bitmapHandle was moved away".

Instead of making this mutable, you need to make all `const UpdateInfo&` receivers as `UpdateInfo&&`.

Reply:
https://github.com/WebKit/WebKit/pull/13418 Addressed there